### PR TITLE
add deal.II publication to main list

### PR DESCRIPTION
**Pull request by tjhei**

### CODE REVIEW

**Comment by tjhei** [comment]
I can add the bibtex entry as well, but I thought I would wait for us to agree on the format.

**Comment by tjhei** [comment]
that's fine with me, we can leave this PR open.

**Comment by tjhei** [comment]
I updated this PR.

Should we move W. Bangerth, R. Hartmann, G. Kanschat down or keep it here?

**Comment by bangerth** [comment]
That's fine by me -- though I think we ought to wait until the citation info for the latest paper is complete with volume and page numbers. It's going to lead to a mess with google scholar if people cite it in different ways.

I do recognize that all of you don't get cited in the meantime, so am flexible with my opinion if you all feel differently.

**Comment by kronbichler** [comment]
I agree that we can wait with moving down the 2007 publication.

File: publications.include [hunk 1]
```diff
@@ -44,6 +44,33 @@
...
+              <br>
+              <pre>
+      @Article{dealii2019design,
+        title   = {The {deal.II} finite element library: design, features, and insights},
+        author  = {Daniel Arndt and Wolfgang Bangerth and Denis Davydov and Timo Heister and Luca Heltai and Martin Kronbichler and Matthias Maier and Jean-Paul Pelteret and Bruno Turcksin and David Wells},
+        journal = {Computers \& Mathematics with Applications},
+        year    = {2020},
+        DOI     = {10.1016/j.camwa.2020.02.022},
+	issn    = {0898-1221},
+        url     = {https://arxiv.org/abs/1910.13247}
```
Comment (bangerth) [inline publications.include:70]: That refers to a different publication and source. Let's take the URL out.

File: publications.include [hunk 2]
```diff
@@ -44,6 +44,33 @@
...
+                  target="_top">preprint</a>)
+              <br>
+              <pre>
+      @Article{dealii2019design,
+        title   = {The {deal.II} finite element library: design, features, and insights},
+        author  = {Daniel Arndt and Wolfgang Bangerth and Denis Davydov and Timo Heister and Luca Heltai and Martin Kronbichler and Matthias Maier and Jean-Paul Pelteret and Bruno Turcksin and David Wells},
+        journal = {Computers \& Mathematics with Applications},
+        year    = {2020},
+        DOI     = {10.1016/j.camwa.2020.02.022},
+	issn    = {0898-1221},
```
Comment (bangerth) [inline publications.include:69]: I think the DOI alone is sufficient.

File: publications.include [hunk 3]
```diff
@@ -44,6 +44,33 @@
...
+              <br>
+              <a href="https://doi.org/10.1016/j.camwa.2020.02.022">DOI: 10.1016/j.camwa.2020.02.022</a>
+              <br>
+	      (<a href="https://arxiv.org/abs/1910.13247"
+                  target="_top">preprint</a>)
+              <br>
+              <pre>
+      @Article{dealii2019design,
+        title   = {The {deal.II} finite element library: design, features, and insights},
+        author  = {Daniel Arndt and Wolfgang Bangerth and Denis Davydov and Timo Heister and Luca Heltai and Martin Kronbichler and Matthias Maier and Jean-Paul Pelteret and Bruno Turcksin and David Wells},
```
Comment (bangerth) [inline publications.include:65]: Since this is in a `<pre>` block, maybe wrap it at 80 characters?

File: publications.include [hunk 4]
```diff
@@ -44,6 +44,33 @@
       }</pre>
             </li>
 
+	    <li>
+              D. Arndt, W. Bangerth, D. Davydov, T. Heister,
+              L. Heltai, M. Kronbichler, M. Maier,
+              J.-P. Pelteret, B. Turcksin, D. Wells
+              <br/>
+              <strong>The <abbr>deal.II</abbr> finite element library: design, features, and insights
```
Comment (bangerth) [inline publications.include:52]:
```suggestion
              <strong>The <abbr>deal.II</abbr> finite element library: Design, features, and insights
```
Same below.

**Comment by bangerth** [comment]
I think we're still waiting for the official publication to happen, but I have some comments that you can already address in the meantime if you want.

**Comment by luca-heltai** [comment]
https://doi.org/10.1016/j.camwa.2020.02.022

**Review by bangerth — APPROVED** [review]
Let's get it merged then!

**Pull request merged**